.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+- '3.6'
+sudo: true
+dist: trusty
+install:
+- sudo sed -i 's/trusty/yakkety/' /etc/apt/sources.list
+- sudo rm -rf /etc/apt/sources.list.d/*
+- sudo apt-get update -qq
+- sudo apt-get install -qq pandoc ca-certificates
+- pip install -r requirements.txt
+script:
+- ":"
+notifications:
+  email: false
+deploy:
+  provider: script
+  script: python g2e.py
+  on:
+    branch: master
+    repo: 3DprintFIT/BI-3DT
+env:
+  global:
+  - secure: pNqZ7oB8D3vnFK+k9Ud4/Vw21NQh7zGn49X9pyi/1qB1xMcSj7rCvN+OiN+myAjQ7DtZYHBdrpF7KMfq+FTDbqKRhEA1MBj7dE3GoScCXNPMTGudZE1k6JfWRvMqEYe5AmGoK4gTupq4ZVq/1wF35SZyvJ5NfoI/BaVlYOFvJeJGUTx56itusWQedJrdZChVcaT4UgwGh7cZokTwysR2upfAJfivkH5cTeNo0yk6aBGpd/lnk+WwjA79K5p1lh6KC0RHwL+ZK0ryeZf1lgEYC6dsqi/KQov23rU2Stb15cRb+n8Zx9jD4FzHQo8dfPTYcWs9E98RdbXYuARrM33ozut60Kmxairm/eDfujtA1/tLpkjmeqiFrsS/v83cLsk6To/bbbKFW2NCH56jM/wMHRKZ3pIPzbor3nonLf+UR5ivfjaUDqVPf02tUT8fab8bFy2XF6jU8MMlDFioMINkc/ATami5MIyyrWu+mAjxn9fdk19vgzQpq+kCdR1TCkmuaO5irGihsrZUxCuuXlk9B5YBjYN+qEDe4qsC0B5Wnym1EtNMSeT6DqrdZFiMWu6xs4F/zdmAj1XZ2/ilp2Au3XzWonank9/sgn1xy5H0fAFkKYV/NJckrw8t3JSwU3ool57yfm+dO6/SkU9sOMYkyG2Y05bunmRq4cVyuWdTuuQ=
```

Comment:
What are these strings?

Reply:
the env used in the script, my Edux credentials :)

Reply:
This is a public repo, get it out of here, you can use Travis web UI to insert secret tokens.

Reply:
this is encrypted :) I use it all the time

Reply:
the only 2 security problmes are:

 * Travis Ci knows my credentials
 * anyone with write access to this repo can do whatever they want with my credentials

(both threads are also present when the web UI is used to set those)

Reply:
as long as someone without write access cannot use the knowledge gain by reading this repo to abuse your creds you are good to go...

Reply:
https://docs.travis-ci.com/user/encryption-keys/